README.md

```diff
@@ -10,22 +10,38 @@ I wanted to make a JavaScript library named pee.js because of a typo and a pun.
 
 ## Usage
 
-### Browser (pee.js)
+This is a hybrid (commonjs and ecmascript module) package that is available on https://www.npmjs.com/package/pee.js
```

Comment:
most Java script packaging is done with npm these days, even for frontend/browser development with tools like webpack and such, there are also many CDNs that will automatically made the npm packages available so that people can include them with `<script>` tags if they want to (such as https://unpkg.com/)